apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -161,10 +161,14 @@ GoogleBlockly.Extensions.register('procedure_def_mini_toolbox', function () {
   flyoutToggleButton.setIcon(false);
 });
 
-// This extension adds an SVG frame around procedures definition blocks.
-// Not used in Music Lab or wherever the modal function is enabled.
+// Adds an SVG frame to procedure definition blocks when they're on the main workspace.
+// Not used in Music Lab, embedded workspaces, or the hidden definition workspace.
```

Comment:
Nit: I think we might mean "modal editor workspace" and not "hidden definition workspace", since the latter isn't even rendered.